src/components/organisms/signupForm/SignupForm.js

```diff
@@ -0,0 +1,28 @@
+import Form from '../../atoms/form'
+import LabelInput from '../../molecules/labelInput';
+import {MailOutlined, LockOutlined} from '../../atoms/icon'
+import Button, { BUTTON_SIZES } from '../../atoms/button'
+import style from './signupForm.module.css';
+import { Input } from 'antd';
+
+const SignupForm = (props) => {
+    const formLayout = {
+        labelCol:{
```

Comment:
what is this for?

Reply:
The main idea was to add a feature for responsive forms : the selected piece of code changes the layout of the form from horizontal to vertical
It isn't used though